getting-started/create-a-site.md

```diff
@@ -0,0 +1,162 @@
+---
+title: How to Set Up a Publishing Site in Curvenote
+short_title: Create a Site
+---
+
+
+
+# Create a Site
+
+:::{tip} ObjectiveCreate a publishing destination for your articles, lab notebooks, or group collections. :::
+
+## Quick Checklist
+
+- Have a Curvenote account & CLI installed
+- Access or create a site (web admin)
+- Configure your project (curvenote.yml)
+- Organize local content (if using Git/GitHub)
+- Add metadata to each file
+- Submit content to a site & collection
+- Push changes for internal review (optional)
+- Run checks before publishing (optional)
+- Publish (admin only)
+
+## Workflow at a Glance
+
+```mermaid
+graph TD
+    A[1. Create Site<br/>web admin] --> B[2. Set Landing Page]
+    B --> C[3. Organize Content<br/>optional for local/Git]
+    C --> D[4. Add Metadata]
+    D --> E[5. Submit Content]
+    E --> F[6. Push Changes<br/>optional]
+    F --> G[7. Run Checks<br/>optional]
+    G --> H[8. Publish<br/>admin only]
+    H --> I[🎉 Site Live!]
+    
+    style A fill:#e1f5fe
+    style I fill:#c8e6c9
+```
+
+**Text Version:**
+
+```
+[1] Create Site (web admin)
+        ↓
+[2] Set Landing Page
+        ↓
+[3] Organize Content (optional for local/Git)
+        ↓
+[4] Add Metadata
+        ↓
+[5] Submit Content → [6] Push Changes (optional)
+        ↓
+[7] Run Checks (optional)
+        ↓
+[8] Publish (admin only)
+        ↓
+    🎉 Site Live!
+```
+ ## Before You Begin
+:::{important} Make sure you have:
+
+- A Curvenote account and Curvenote CLI installed
+- Access to an existing site (or permission to create one via the web admin interface)
+- A configured project (curvenote.yml) if publishing from GitHub or locally
+:::
+
+
+### 1. Create a Site (via Web Admin)
```

Comment:
I'd recommend removing the hard-coded numbers in all of these headings (across all the pages). We can turn on and off numbering based on heading level. Then, if we add/remove sections, the numbering is updated automatically.